### PR TITLE
Brand and showcase thumbnail links

### PR DESCRIPTION
**Pull request by Stormheg**
This PR will add the ability to add a link to a brand logo image on the homepage. It also makes the thumbnails in the showcase section part of the link.

### CODE REVIEW

File: wagtailio/core/templates/core/blocks/brands_block.html [hunk 1]
```diff
@@ -2,17 +2,26 @@
...
 
             <div class="logo-list__carousel carousel">
-                {% for brand in value.brands %}
-                    {% image brand width-300 as brand_image %}
-                    <div><img src="{{ brand_image.url }}" alt="{{ brand.title }}" /></div>
+                {% for brand in value.brands2 %}
+
+                    {% image brand.image width-300 as brand_image %}
+                    {% if brand.url %}
+                        <a href="{{ brand.url }}">
```
Comment (kaedroho) [inline wagtailio/core/templates/core/blocks/brands_block.html]: If these are external links, should we add `target="_blank"`?

File: wagtailio/core/templates/core/blocks/brands_block.html [hunk 2]
```diff
@@ -2,17 +2,26 @@
...
+                        <img src="{{ brand_image.url }}" alt="{{ brand_image.title }}" />
+                        </a>
+                    {% else %}
+                    <div>
```
Comment (kaedroho) [inline wagtailio/core/templates/core/blocks/brands_block.html]: Some inconsistency with indentation here and in the `` tag above

File: wagtailio/core/templates/core/blocks/brands_block.html (hunk 1 above)
Comment (Stormheg) [reply wagtailio/core/templates/core/blocks/brands_block.html]: I think we need that yes! 👍

Comment (gasman) [reply wagtailio/core/templates/core/blocks/brands_block.html]: No, don't do it! https://github.com/wagtail/wagtail/issues/1167#issuecomment-199594073

**Comment by Stormheg** [comment]
closing, superseded by #44

**Pull request closed without merging**